wemake_python_styleguide/visitors/ast/naming.py

```diff
@@ -79,10 +78,7 @@ def visit_Attribute(self, node: ast.Attribute) -> None:
 
         self.generic_visit(node)
 
-    def visit_FunctionDef(
-        self,
-        node: Union[ast.FunctionDef, ast.AsyncFunctionDef],
-    ) -> None:
+    def visit_FunctionDef(self, node: MethodMembers) -> None:
```

Comment:
One last thing. This is not quite clean to declare a `FunctionDef` handler that also visits `AsyncFunctionDef`. It violates the design idea of `ast.NodeVisitor`.

What can be done instead? We can create a proxy handler called `visit_any_function` and alias it to support both function types.
Example link: https://github.com/wemake-services/wemake-python-styleguide/blob/5539d0e873db7dc7b841fc0242b23b1fa329af7a/wemake_python_styleguide/visitors/ast/complexity/counts.py#L89-L92

Reply:
My bad, did not notice... Fixed.